src/static/webpack/index.js

```diff
@@ -83,6 +83,7 @@ export async function startDevServer ({ config }) {
     quiet: true,
     watchOptions: {
       ignored: /node_modules/,
+      ...(config.devServer.watchOptions || {})
     },
     ...config.devServer,
```

Comment:
`watchOptions` needs to be moved *after* the `...config.devServer` part. This way it won't get overridden a second time around.